perfkitbenchmarker/linux_packages/aerospike_server.py

```diff
@@ -19,13 +19,13 @@
 
 from perfkitbenchmarker import data
 from perfkitbenchmarker import flags
-from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import INSTALL_DIR
 
 FLAGS = flags.FLAGS
 
 GIT_REPO = 'https://github.com/aerospike/aerospike-server.git'
 GIT_TAG = '3.7.5'
-AEROSPIKE_DIR = '%s/aerospike-server' % vm_util.VM_TMP_DIR
+AEROSPIKE_DIR = '%s/aerospike-server' % INSTALL_DIR
```

Comment:
The code seems to be split about 50/50. You'll notice that most of the places where I changed vm_util.VM_TMP_DIR to INSTALL_DIR use format strings.

I don't mind changing everything to posixpath.join, but I'd prefer to do it as a follow-on commit. I'll change everything, including the stuff that still uses VM_TMP_DIR. I'm also willing to change everything to use format strings -- that'd be my preference since posixpath.join() is just as portable as '%s/%s', but I'm not picky.